github-for-jira.sd.yml

```
@@ -1,4 +1,4 @@
-buildNumber: 0.0.0
+buildNumber: 0.0.3
```

This should always stay `0.0.0` ... the deployment pipeline always deploys this tag
```suggestion
buildNumber: 0.0.0
```